aiohttp/web_response.py

```diff
@@ -355,11 +359,14 @@ def _start(self, request,
                 del headers[CONTENT_LENGTH]
         elif self._length_check:
             writer.length = self.content_length
-            if writer.length is None and version >= HttpVersion11:
-                writer.enable_chunking()
-                headers[TRANSFER_ENCODING] = 'chunked'
-                if CONTENT_LENGTH in headers:
-                    del headers[CONTENT_LENGTH]
+            if writer.length is None:
+                if version >= HttpVersion11:
+                    writer.enable_chunking()
+                    headers[TRANSFER_ENCODING] = 'chunked'
+                    if CONTENT_LENGTH in headers:
```

Comment:
These two lines cannot be covered in current version because all the **content_length** implementations do not return None when there is a Content-Length header. But I think it is necessary in case there are other implementations in the future. We should not rely on the current implementations.